upload/src/addons/SV/WarningImprovements/Listener.php

```diff
@@ -61,4 +61,20 @@ public static function entityPreSave_XFWarningDefinition(\XF\Mvc\Entity\Entity $
 
     public static $customWarningPhrase_version_id = 101011;
     public static $customWarningPhrase_version_string = '1.1.1s';
+
+    public static $warningActionData = '';
+
+    public static function entityPreSave_XFWarningAction(\XF\Mvc\Entity\Entity $entity)
```

Comment:
You have already extending the Warning Action entity, move this to _preSave() method.

Reply:
Done